docs/setup/build.md

```diff
@@ -1,60 +1,33 @@
-# Cross Compiling KubeEdge
+# Build from source
 
-In most of the cases, when you are trying to compile KubeEdge edgecore on Raspberry Pi or any other device, you may run out of memory, in that case, it is advisable to cross-compile the Edgecore binary and transfer it to your edge device.
```

Comment:
`out of memory` is needed here for fresh developers.

Reply:
Done.